test/utils.jl

```diff
@@ -0,0 +1,139 @@
+module TestUtils
+    using StatsBase
+    using Base.Test
+    using NullableArrays
+
+    @testset "describe" begin
+        io = IOBuffer()
+        describe(io, collect(1:10))
+        @test String(take!(io)) == """
+                                   Summary Stats:
+                                   Mean:         5.500000
+                                   Minimum:      1.000000
+                                   1st Quartile: 3.250000
+                                   Median:       5.500000
+                                   3rd Quartile: 7.750000
+                                   Maximum:      10.000000
+                                   """
+        describe(io, ["s"])
```

Comment:
Sanity checks for method ambiguity errors. Having confirmed there aren't any, the most significant thing they add is probably CI runtime. I've take them out. Thanks!